deploy/clowdapp.yaml

```diff
@@ -180,16 +186,19 @@ parameters:
   - description: Instance prefix adds string to all instance names, leave blank for production
     name: APP_INSTANCE_PREFIX
     value: ""
+  - description: Application cache type (none, memory or redis)
+    name: APP_CACHE_TYPE
+    value: "redis"
   - description: Internal queue type (memory/sqs/postgres).
     name: WORKER_QUEUE
-    value: "memory"
```

Comment:
memory's gone? :)

Reply:
What do you mean, the value is set on line `194` below to "redis".

Reply:
ah, I've missread it :(